apps/constellation/main.cpp

```diff
@@ -252,15 +270,13 @@ int main(int argc, char **argv)
       // attempt to build the configuration for constellation
       fetch::constellation::Constellation::Config cfg = BuildConstellationConfig(settings);
 
+      // setting policy for critical signals
+      shutdown_on_critical_failure = settings.shutdown_on_critical_failure.value();
+
       // create the bootrap monitor (if configued to do so)
       auto initial_peers = ToUriSet(settings.peers.value());
       auto bootstrap     = CreateBootstrap(settings, cfg, p2p_key, initial_peers);
 
-      for (auto const &uri : initial_peers)
```

Comment:
Please don't remove this - it is useful for debugging what is going on with bootstrap - which raises additional questions